Resources/Locale/en-US/ninja/role.ftl

```diff
@@ -0,0 +1,7 @@
+ninja-role-greeting =
+    I am an elite mercenary of the mighty Spider Clan!
+    Surprise is my weapon. Shadows are my armor. Without them, I am nothing.
+
+ninja-role-greeting-direction =
+    The station is located to your {$direction} at {$position}.
+    A thrown ninja soap will be a great way to get there.
```

Comment:
Soap's a creative way to get there, but the Ninja should spawn with a jetpack if this is going to be an issue.

Reply:
soap isn't just for getting to station it's mainly for slipping people in a chase since ninja shoes are noslip. you can also use katana dash to teleport to the station

and fun fact in /tg/ old ninja did have a jetpack

Reply:
I still think it would be better if you gave Ninja a jetpack instead of a backpack. I've played on servers where they didn't have the jetpack and on ones where they did, and it was way worse trying to navigate your way to the station with just dashes and stars compared to having the jetpack. Please give them a jetpack on spawn if they're spawning in space.

Reply:
then they cant get any tools which really sucks for more nuanced sabotaging